archetypes/example-kar-plugin/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,33 @@
+# OpenNMS ${pluginName} Plugin
+
+## Overview
+
+This plugin was generated by the OIA archetype and is used as example project layout.
+
+## Developing
+
+Build and install the plugin into your local Maven repository using:
+```
+mvn clean install
+```
+
+> OpenNMS normally runs as root, so make sure the artifacts are installed in `/root/.m2` or try making `/root/.m2` symlink to your user's repository
+
+From the OpenNMS Karaf shell:
+```
+feature:repo-add mvn:${groupId}/karaf-features/1.0.0-SNAPSHOT/xml
+feature:install opennms-plugins-${pluginId}
+```
+
+Update automatically:
+```
+bundle:watch *
+```
+
+## Features
+
+Once installed, the plugin makes the following Karaf shell commands available:
+* opennms-${pluginId}:stats
+* opennms-${pluginId}:push-topology
+
+You can also access the REST endpoint mounted by the plugin at `http://nms:8980/opennms/rest/${pluginId}/ping`
```

Comment:
I wonder if it shouldn't be _localhost_ instead of _nms_? Most developers will run the system locally?

Reply:
Done.